### PR TITLE
BUG: Fix introduction episode non-solution notebook

### PR DESCRIPTION
**Pull request by jhlegarreta**
Fix introduction episode non-solution notebook: use the correct dataset
and subject for the analyses.

Fixes:
```
ValueError: BIDS root does not exist: /home/runner/work/SDC-BIDS-dMRI/SDC-BIDS-dMRI/data/ds000030
```

raised in
https://github.com/carpentries-incubator/SDC-BIDS-dMRI/runs/3025093598?check_suite_focus=true#step:7:44

Synchronize the markdown episode contents and the Jupyter notebooks
(both the non-solution and solution).

Take advantage of the commit to:
- Fix the static image relative path.
- Fix the dataset tree: change the duplicate `fmap` folder for the
missing `func` folder.
- Add the `bids` initial dot extension config statement to avoid the
related warning being displayed.
- Remove the `dipy` and `fury` installation steps: both tools are
assumed to have been installed during setup.
- Use the same DWI volume for to be displayed.
- Fix the output of the bvecs.
- Removed the unexplained `b0` volume slicing block.
- Add exercise to notebooks.
- Fix the suffix syntax in exercise.
- Remove the second exercise: it is already explained/done at the
beginning of the episode.
- Fix typos.
- Remove outputs from non-solution notebook.

### CODE REVIEW

**Comment by jhlegarreta** [comment]
Partially fixes #14 (addresses the third task).

I ignore if the `osf` copy of the data is a faithful mirror of the `openneuro` data; i.e. if the treee would look as it is written. I sticked to what I saw in `openneuro`. Please check and amend the commit as necessary.

I'd say that I'd be better to merge this prior to #165, since the latter's changes can more easily be applied if the rebasing derives in hard-to-solve conflicts.

**Comment by kaitj** [comment]
Looks fine to me, I don't recall if the subject is still included in `ds000221`, but the structure would still be the same (just different subject id). This seems like a good intermediate step - if i recall @josephmje was working on a revamped intro (or was that already merged in)?

File: _episodes/introduction.md [hunk 1]
```diff
@@ -122,18 +129,23 @@ tree '../data/ds000221'
...
-Lets first pull the metadata from its associated JSON file using the
+Let's first pull the metadata from its associated JSON file using the
 <code>get_metadata()</code> function for the first run.
 
 ~~~
+import bids
 from bids.layout import BIDSLayout
 
+?BIDSLayout
+
```
Comment (josephmje) [inline _episodes/introduction.md:140]:
```suggestion
```

**Review by josephmje — APPROVED** [review]

File: _episodes/introduction.md (hunk 1 above)
Comment (jhlegarreta) [reply _episodes/introduction.md:140]: I left it because it shows the in-notebook help of the class, and thought that it was useful.

**Comment by jhlegarreta** [comment]
Further improvements can come in separate PRs. This would be a step towards making the episode and its notebooks consistent, and hopefully have the tests passing, since the non-solution notebook is non-functional as it is now.

**Pull request merged**